docs/demo/data/processed/model_extensions.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec 19 10:50:55 2023
+
+@author: bdobson
+"""
+from wsimod.core import constants
+import sys
+import os
+from tqdm import tqdm
+from math import log10
+from wsimod.nodes.land import ImperviousSurface
+from wsimod.nodes.nodes import QueueTank, Tank, ResidenceTank
+
+def extensions(model):
+    # Apply customations
+    model.nodes['my_groundwater'].residence_time *= 2
```

Comment:
Based on my experience, it can be done via the input file but then the user needs to define the new type of nodes, which requires more efforts in naming and tidying everything.